pkg/apis/certificates/v1alpha1/conversion.go

```diff
@@ -16,9 +16,24 @@ limitations under the License.
 
 package v1alpha1
 
-import "k8s.io/kubernetes/pkg/runtime"
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
+)
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
 	// Add non-generated conversion functions here. Currently there are none.
```

Comment:
comment is no longer relevant. feel free to reapply if you get around to it, or fix up later.